lib/modules.txt

```diff
@@ -0,0 +1,43 @@
+require('assert');
```

Comment:
what is the purpose of this file and how does it get generated and should it be committed?